ui/src/services/utils.js

```diff
@@ -329,3 +329,26 @@ export const fromUnixTimestampToDate = (date) => {
     return new Date(date * 1000);
   }
 };
+
+// Convert the timestamp in milliseconds to an object with day, hour, minute and second
+export const fromMilliSectoAge = (milliSecTime) => {
+  if (milliSecTime && milliSecTime > 0) {
```

Comment:
The first `milliSecTime` check is useless.

Reply:
Indeed!